taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -40,27 +39,38 @@ const SearchResults = ({query}) => {
     // Helper functions for Dialog component
     function handleCloseAddDialog() { setOpenAddDialog(false); }
     function handleOpenAddDialog() { setOpenAddDialog(true); }
-    function handleCloseSuccessDialog() { setOpenSuccessDialog(false); }
-    function handleOpenSuccessDialog() { setOpenSuccessDialog(true); }
+    function handleOpenSuccessSnackbar() { setOpenSuccessSnackbar(true); }
+    function handleCloseSuccessSnackbar() { setOpenSuccessSnackbar(false); }
     
     function handleAddNode() {
         const newNodeID = newLanguageCode + ':' + newNode // Reconstructing node ID
         const data = {"id": newNodeID, "main_language": newLanguageCode};
-        fetch(url, {
+        fetch(API_URL+'nodes', {
             method : 'POST',
             headers: {"Content-Type" : "application/json"},
             body: JSON.stringify(data)
         }).then(() => {
             handleCloseAddDialog();
-            handleOpenSuccessDialog();
+            handleOpenSuccessSnackbar();
         }).catch((errorMessage) => {
             // Do nothing
         })
     }
 
     // Displaying errorMessages if any
     if (isError) {
-        return (<Typography variant='h5'>{errorMessage}</Typography>)
+        return (
+            <Container component="main" maxWidth="xs">
+            <Grid
```

Comment:
why using a Grid with a single component?

Reply:
Oh, I see that it is kind of for ui consistency as also used in the success. Amm, may be ok, although I'd try anyways to center without the grid

Reply:
Ah yep, I used the Grid to center the components and the error messages. The `Container` component has been removed.